### PR TITLE
Add bar height option (Airblader/i3)

### PR DESCRIPTION
**Pull request by mikkeloscar**
Make it possible to set the bar height.

### CODE REVIEW

File: sway/commands.c [hunk 1]
```diff
@@ -1592,6 +1593,27 @@ static struct cmd_results *bar_cmd_colors(int argc, char **argv) {
...
+				"Invalid height value: %s", argv[0]);
+	}
+
+	config->current_bar->height = height;
+	sway_log(L_DEBUG, "Setting bar height to %d on bar: %s", height, config->current_bar->id);
+	return cmd_results_new(CMD_SUCCESS, NULL, NULL);
+
+
+
+	return cmd_results_new(CMD_SUCCESS, NULL, NULL);
```
Comment (SirCmpwn) [inline sway/commands.c]: Uhh?

**Comment by mikkeloscar** [comment]
Fixed double return and renamed `bar_height` to `height` in `sway/ipc-server.c`

File: sway/ipc-server.c [hunk 2]
```diff
@@ -419,7 +419,7 @@ void ipc_client_handle_command(struct ipc_client *client) {
 			}
 			json_object_object_add(json, "status_command", json_object_new_string(bar->status_command));
 			json_object_object_add(json, "font", json_object_new_string(bar->font));
-			json_object_object_add(json, "bar_height", json_object_new_int(bar->bar_height));
+			json_object_object_add(json, "height", json_object_new_int(bar->height));
```
Comment (SirCmpwn) [inline sway/ipc-server.c]: You can't change this JSON property name - it's backwards compatible with i3bar.

**Comment by mikkeloscar** [comment]
Sorry about that, I have reverted JSON property name to `bar_height`.

**Pull request merged**